test_resources/load-durations.edn

```diff
@@ -33,6 +29,10 @@
  ["test-data" "checkins"]                                             130413900
  ["test-data" "users"]                                                23762400
  ["test-data" "venues"]                                               29380500
+ ["test-data" "orders"]                                               3439047900
+ ["test-data" "people"]                                               1922302400
+ ["test-data" "products"]                                             95636200
+ ["test-data" "reviews"]                                              292738400
```

Comment:
These load durations aren't right anymore, as they are from when they were in the `sample-dataset`. 

But I moved them to `test-data` for now.